src/backend/app.js

```diff
@@ -21,15 +23,129 @@ app.use(cors());
 
 router.use("/meals", mealsRouter);
 
+//----------------------------------------------------------------------------------------------------------------------
+// Respond with all meals in the future (relative to the when datetime) - /future-meals
+
+app.get('/future-meals', async (req, res) => {
+  try {
+    const [rows] = await knex.raw("SELECT meal.id, meal.title, meal.meal_date_time FROM `meal` WHERE `meal_date_time` >= CURDATE()")
+    if ([rows]) {
+      res.json({
+        status: 'success',
+        data: [rows]
```

Comment:
No reason to wrap rows inside array. You can just return `data: rows`

Reply:
Made suggested changes.